internal/builders/gradle/collect_release_artifacts.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
```

Comment:
Make sure to fail when any command fails.
```suggestion

set -euo pipefail

```